x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/cache/shared/PartiallyCachedShardAllocationIntegTests.java

```diff
@@ -330,8 +329,6 @@ public void onFailure(Exception e) {
             Math.abs(shardCountsByNodeName.get(newNodes.get(0)) - shardCountsByNodeName.get(newNodes.get(1))),
             lessThanOrEqualTo(1)
         );
-        logger.info("--> shardCountsByNodeName {}, new nodes {}", shardCountsByNodeName, newNodes);
```

Comment:
I thought this was needed for debugging :D

Reply:
This is from another issue, I think it is easier to reproduce it locally, though I am not sure what causing it yet.